scripts/ci-e2e.sh

```diff
@@ -37,14 +37,10 @@ ARCH=$(uname -m)
 PVSADM_VERSION=${PVSADM_VERSION:-"v0.1.4-alpha.3"}
 E2E_FLAVOR=${E2E_FLAVOR:-}
 REGION=${REGION:-"us-south"}
-RESOURCE_GROUP=${RESOURCE_GROUP:-"prow-resource-group"}
 
 trap cleanup EXIT
 
 cleanup(){
-    # Delete the created ports for the network instance
-    [ -n "${NEW_PORT}" ] && ./pvsadm delete port --network ${IBMPOWERVS_NETWORK_NAME} --port-id ${PORT_ID} --instance-id ${IBMPOWERVS_SERVICE_INSTANCE_ID}
```

Comment:
why are we not deleting the port anymore?

Reply:
As the test uses resources of type powervs service instance, we are depending on the ibmcloud-janitor to take care of the cleanup of ports, network and instances.